packages/widgetbook/lib/src/fields/iterable_segmented_field.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import '../../widgetbook.dart';
+
+/// A [Field] that builds [SegmentedButton]<[Iterable<T>]> for [Iterable] values.
+class IterableSegmentedField<T> extends Field<Iterable<T>> {
```

Comment:
This new field should be documented [here](https://github.com/earminjon/widgetbook/blob/71ee912082949746393855fa80d4ec81e6e8159e/docs/essentials/fields.mdx#L8-L21).